tests/crs.R

```diff
@@ -41,7 +41,15 @@ if (sf_extSoftVersion()["USE_PROJ_H"] == "true" || sf_proj_info("have_datum_file
   "datum files not installed"
 }
 
-x = st_crs(sp::CRS("+proj=longlat +ellps=WGS84 +no_defs"))
+x0 = sp::CRS(SRS_string="EPSG:4326")
```

Comment:
This fails when the GDAL/PROJ versions are too old (see CI: ubuntu 16:04 release). The breaking vignettes on old-rel are also worrying, just not informative where things go wrong.

Reply:
CI now tells me a very high noise/signal ratio. I cannot see the real R CMD check output anywhere. I've reverted to `"+init=epsg:4326"`.